quetz/authentication/base.py

```diff
@@ -109,7 +109,7 @@ async def authorize(
         user_dict = await self._authenticate(request, dao, config)
 
         if user_dict is None:
-            return Response("login failed")
+            return Response("login failed", status_code=401)
```

Comment:
@wolfv, this change affects all authenticators, I believe. I think previously, it would always return `200`, even when the login failed. This is not strictly necessary in this PR, so if you want to can remove it/discuss it in a separate PR.

Reply:
I'd suggest to leave this out for now and open a separate PR for it. Will probably make it easier to get this PR merged.